Makefile

```diff
@@ -0,0 +1,14 @@
+VERSION := `head -1 debian/changelog | awk '{print $2}' | grep -o '\d\+.\d\+'`
```

Comment:
Ah yes. The version of grep does matter. I have updated it to your command as it is more robust